components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -487,6 +487,218 @@ paths:
           -H "Content-Type: application/json" -d @data.json
           "https://127.0.0.1:9443/api/am/publisher/v4/apis/7a2298c4-c905-403f-8fac-38c73301631f/topics"'
 
+  /apis/{apiId}/custom-backend/{customBackendId}/content:
```

Comment:
rename to sequence-backend/{sandbox/Prod}/{sequenceID} check dev portal key type implementation